tests/components/relatedProductsTests/AddCard.test.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import React from 'react'
+import 'regenerator-runtime/runtime'
```

Comment:
If you are not using async operations, we can remove the regenerator-runtime import

Reply:
Will do!